sys/include/stdio_base.h

```diff
@@ -34,6 +34,18 @@ extern "C" {
  */
 void stdio_init(void);
 
+/**
+ * @brief Get number of bytes available for reading.
+ *
+ * @return nr of available bytes
+ */
+int stdio_avail(void);
```

Comment:
This works only, if all `stdio_*` implementations support this function. `std_uart` does not at the moment.